awscli/examples/application-signals/untag-resource.rst

```diff
@@ -0,0 +1,13 @@
+**To remove one or more tags from the specified resource**
+
+The following ``untag-resource`` example removes one or more tags from the specified resource. ::
+
+    aws application-signals untag-resource \
+        -—resource-arn "arn:aws:application-signals:us-east-1:123456789101:slo/SLOName" \
+        -—tag-keys "test"
+    
+Output::
```

Comment:
```suggestion
```

Reply:
Can you remove the `Output::` line from the commands that don't have any. It's actually invalid .rst, hence the test failures.